README.md

```diff
@@ -64,27 +92,7 @@ started contributing, please see our [Contributor Guide](CONTRIBUTING.md).
 <!-- More information about creating your scope with links to examples -->
 <!-- https://contribute.cncf.io/maintainers/governance/charter/ -->
 
-### In Scope
-
-[TODO: PROJECTNAME] is intended to [TODO: Core functionality]. As such, the
-project will implement or has implemented:
-
-- [TODO: High-level Item 1]
-- [TODO: High-level Item 2]
-- [TODO: High-level Item 3]
-
-### Out of Scope
-
-[TODO: PROJECTNAME] will be used in a cloud native environment with other
-tools. The following specific functionality will therefore not be incorporated:
-
-- [TODO: Excluded function 1]
-- [TODO: Excluded function 2]
-
-[TODO: PROJECTNAME] implements [TODO: List of major features, existing or
-planned], through [TODO: Implementation
-requirements/language/architecture/etc.]. It will not cover [TODO: short list
-of excluded items]
+- Define a set of query interfaces that can standardized across various software ecosystems.
```

Comment:
Missing `### In Scope` section header

Reply:
I removed that for now. Once things are more clear, we can add that again (at least that is my plan)
Less clutter!

Reply:
Let me know if you or others disagree.